docs/data_node/extract_node/doris.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Doris
+sidebar_position: 11
+---
+
+import {siteVariables} from '../../version';
+
+## Overview
+
+`Doris Extract` node supports reading data from Doris. This chapter describes how to set up a Doris Extract
+node to run SQL queries against the Doris database.
+
+## Supported Version
+
+| Extract Node	       | Doris version  |                                                                                                                                                                                                                                                                                                                                                                                           
+|---------------------|----------------|
+| [Doris](./doris.md) | 0.13+          |  
+
+## Dependencies
+
+In order to set up the Doris Extract node, the dependency information needed to use build automation tools
+such as Maven or SBT is provided below.
+
+### Maven dependency
+
+<pre><code parentName="pre">
+{`<dependency>
```

Comment:
![image](https://user-images.githubusercontent.com/107393625/189802975-3317e21b-4016-4065-94b5-bd1bf24b1c2e.png)
wrong format

Reply:
Please correct the above format errors in the Chinese document in the same time.